File: Python/Program/Cpu.py
from Komponen import Komponen

class Cpu(Komponen):
    def __init__(self, merk, nama, jumlahCore, kecepatanGHz):
        super().__init__(merk, nama)
        self.jumlahCore = jumlahCore
        self.kecepatanGHz = kecepatanGHz
        
    def setJumlahCore(self, jumlahCore):
        self.jumlahCore = jumlahCore
        
    def getJumlahCore(self):
        return self.jumlahCore
    
    def setKecepatanGHz(self, kecepatanGHz):
        self.kecepatanGHz = kecepatanGHz
        
    def getKecepatanGHz(self):
        return self.kecepatanGHz

File: Java/Program/Cpu.java
public class Cpu extends Komponen 
{
    private int jumlahCore;
    private double kecepatanGHz;

    public Cpu(String merk, String nama, int jumlahCore, double kecepatanGHz) 
    {
        super(merk, nama);
        this.jumlahCore = jumlahCore;
        this.kecepatanGHz = kecepatanGHz;
    }

    public void setJumlahCore(int jumlahCore) 
    {
        this.jumlahCore = jumlahCore;
    }

    public int getJumlahCore() 
    {
        return jumlahCore;
    }

    public void setKecepatanGHz(double kecepatanGHz) 
    {
        this.kecepatanGHz = kecepatanGHz;
    }

    public double getKecepatanGHz() 
    {
        return kecepatanGHz;
    }
}